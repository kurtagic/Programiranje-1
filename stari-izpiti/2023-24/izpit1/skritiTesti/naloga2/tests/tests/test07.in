1 52 64 927
13 50 94
13 50 13
13 50 443
13 50 687
13 50 471
13 50 79
13 50 552
13 50 220
0 56 395
0 56 363
0 56 264
0 50 411
0 56 636
13 25 306
13 56 85
0 50 698
13 50 29
13 56 709
13 50 241
45 25 556
13 25 671
0 50 267
34 56 570
45 16 462
41 50 522
13 16 41
41 25 338
45 62 121
13 50 16
50 16 533
50 62 70
25 56 25
50 62 113
34 56 471
41 16 266
34 56 780
34 16 151
50 56 25
34 62 643
45 57 727
25 39 220
25 51 671
13 57 446
13 25 12
34 25 43
34 62 775
41 16 694
41 62 673
34 50 280
34 56 52
45 57 696
25 16 505
0 62 647
25 55 355
25 51 780
13 39 514
50 50 151
34 56 474
50 56 17
13 50 489
45 57 108
50 56 207
50 25 265
23 50 387
5 57 445
13 16 730
23 62 8
50 57 649
13 63 183
5 25 437
50 62 93
0 25 567
23 16 229
50 51 9
25 16 705
34 25 581
34 16 452
41 63 554
5 56 608
25 16 471
25 57 679
23 51 722
3 57 556
25 39 442
3 50 464
25 50 691
0 56 686
5 63 541
50 56 666
13 63 765
25 56 264
16 50 319
50 50 593
5 16 722
5 25 715
3 25 586
7 16 551
34 63 482
7 50 147
34 51 626
2 51 446
50 57 635
23 50 257
16 16 447
25 56 730
0 16 186
13 25 694
43 39 708
50 62 614
39 63 300
17 34 433
3 63 790
5 39 681
2 56 46
23 63 58
43 56 301
13 8 247
5 57 70
16 39 161
39 55 117
34 51 616
39 63 16
5 17 649
50 8 431
0 7 642
3 50 602
44 56 521
25 56 725
5 50 192
13 51 369
23 63 596
5 57 115
34 56 616
17 39 408
7 25 70
23 56 367
5 39 786
45 14 69
5 34 285
43 17 409
5 63 579
43 56 738
39 56 151
7 51 609
5 62 791
7 17 266
3 7 717
47 56 442
34 7 232
34 25 695
50 14 669
25 16 165
43 7 117
7 7 630
45 16 475
50 62 328
45 57 482
44 34 210
39 51 23
3 56 306
3 7 200
3 51 261
23 63 409
3 28 227
34 28 41
5 39 545
23 40 65
34 7 493
5 55 580
47 8 207
13 28 87
39 50 768
7 25 139
3 25 206
17 39 712
25 25 629
34 28 384
13 56 478
5 63 704
3 55 412
23 28 275
41 62 8
3 34 243
39 16 491
12 25 358
25 40 226
1 39 28
45 25 64
23 34 319
27 28 299
39 55 47
25 7 102
13 63 297
3 56 191
13 16 277
25 25 784
45 57 565
23 25 566
45 8 533
23 25 752
39 51 371
23 55 587
26 33 256
3 56 642
23 19 324
23 22 706
2 56 268
17 56 247
34 7 510
23 34 598
23 8 71
17 7 723
5 62 432
25 28 790
26 16 240
2 7 491
44 55 781
17 57 518
3 22 114
23 16 390
2 16 25
23 22 309
5 56 548
2 33 152
44 28 114
50 25 103
25 55 555
1 51 315
1 57 378
45 7 397
37 33 277
3 22 590
2 51 505
2 28 339
0 55 391
38 7 174
38 56 622
32 55 789
0 63 123
39 62 81
44 56 558
23 8 519
17 14 348
7 56 503
33 8 489
44 40 751
47 50 694
13 56 731
39 7 713
50 6 535
2 46 605
25 47 759
46 7 769
17 57 192
5 56 120
44 6 91
2 62 634
0 55 369
23 28 677
16 14 703
0 19 157
45 25 704
27 47 587
37 33 63
23 7 629
2 7 677
44 56 248
2 63 338
44 14 107
27 19 486
37 60 739
17 22 199
23 55 369
27 28 698
3 62 450
2 51 289
1 56 130
25 22 500
37 44 698
23 33 299
12 47 786
38 56 366
47 26 526
13 46 478
3 33 53
10 41 319
7 7 309
25 14 367
23 7 297
50 55 357
2 44 278
3 25 247
33 33 786
23 33 557
43 22 53
33 25 61
1 51 222
12 42 8
1 39 668
10 32 425
41 59 22
23 41 109
5 44 309
35 55 494
2 32 334
0 60 480
10 56 368
3 13 416
26 44 577
7 55 676
0 50 98
45 22 785
23 44 761
5 46 703
33 28 227
43 28 460
2 16 1
32 22 539
41 62 98
3 22 712
23 51 115
50 55 284
47 14 622
3 59 328
33 20 685
39 28 11
45 22 392
32 47 399
44 7 488
26 6 417
45 33 613
23 39 397
0 51 87
3 51 277
24 55 487
23 60 369
3 56 482
23 39 410
38 46 528
0 57 125
23 28 396
2 7 431
33 47 366
43 14 658
5 40 215
10 26 233
17 29 788
43 44 128
47 13 337
45 28 97
0 55 295
23 56 631
3 3 87
46 56 515
47 39 80
28 46 160
17 44 225
44 51 376
2 51 336
43 63 78
5 56 66
43 25 334
23 46 740
38 33 774
47 37 551
32 39 185
17 51 133
24 13 252
23 40 740
34 29 232
17 6 578
0 14 562
37 13 176
35 25 544
41 17 660
16 33 475
10 17 401
5 51 744
10 37 134
3 57 467
43 57 446
2 60 190
6 63 447
2 53 325
47 44 543
50 46 67
28 40 720
41 56 295
12 46 455
23 40 673
34 32 131
37 63 740
17 19 51
45 46 207
0 25 97
44 20 667
33 22 604
0 10 25
35 39 164
2 20 316
24 7 37
45 39 704
33 22 297
38 44 84
5 51 481
35 29 483
0 34 602
32 22 167
7 20 329
50 63 161
17 56 61
32 62 601
33 28 193
7 14 108
7 55 378
25 63 338
7 40 461
5 62 361
0 55 95
46 46 286
45 11 143
25 14 574
34 3 210
33 63 510
41 63 487
10 50 38
35 19 761
34 34 532
34 40 168
44 46 629
17 28 301
26 2 76
2 7 754
12 50 120
17 17 32
10 10 422
38 50 670
5 53 534
26 34 250
4 56 366
3 51 403
23 34 384
3 16 194
47 51 757
1 46 663
12 34 345
45 10 568
17 6 766
5 17 267
23 59 196
27 39 542
10 16 130
37 1 60
38 25 457
17 56 142
7 26 543
26 29 41
45 17 662
43 7 250
50 14 784
0 37 586
12 18 164
33 10 108
24 40 439
26 16 83
46 32 692
37 8 29
45 8 276
5 56 766
32 60 389
46 63 526
3 2 37
45 3 497
2 18 306
10 8 169
23 7 162
0 51 249
6 28 206
1 16 404
23 63 19
37 25 126
33 2 309
12 51 635
3 28 744
47 10 679
3 37 584
27 20 354
51 53 3
2 39 90
36 41 200
0 39 95
36 16 220
1 14 277
25 28 60
16 11 348
27 59 347
12 28 790
32 20 185
27 39 155
4 39 16
25 63 663
36 39 132
44 14 630
41 14 568
0 32 413
23 63 786
43 34 225
2 44 54
12 7 87
38 11 334
38 2 589
3 39 333
3 22 191
25 35 783
6 1 462
1 23 79
0 57 199
33 51 241
5 16 109
1 13 329
32 57 145
23 7 206
39 20 581
38 41 478
43 59 345
37 57 208
0 42 371
7 16 702
41 23 581
6 19 107
47 47 421
37 39 193
51 32 569
27 6 269
33 17 118
36 22 160
46 60 432
33 6 52
7 37 164
23 52 326
3 51 282
33 32 397
26 28 138
28 11 606
23 55 96
24 56 319
41 42 343
37 1 425
2 2 327
4 10 340
47 42 363
47 25 64
41 62 596
38 26 260
12 13 724
12 47 723
17 34 151
4 47 33
26 13 226
37 7 553
23 13 624
9 14 252
47 8 631
28 1 595
33 25 371
34 14 616
28 28 634
36 25 32
41 46 674
2 7 390
33 60 126
27 11 196
43 31 204
45 11 14
35 6 544
38 12 662
1 42 3
26 26 51
13 23 551
24 34 394
3 53 376
51 29 313
23 11 571
35 59 732
23 22 137
43 16 485
17 23 713
5 46 430
33 12 308
45 37 529
41 51 36
33 52 277
7 8 279
51 49 374
28 46 98
2 16 10
34 20 241
0 56 494
13 59 637
24 51 682
7 23 433
36 44 278
44 33 672
50 41 741
7 55 330
28 51 65
37 60 755
26 5 323
4 35 559
34 20 516
41 23 141
5 32 504
7 50 634
26 51 346
24 56 550
2 22 8
2 3 573
0 23 785
51 34 352
17 16 314
13 31 397
9 25 6
7 40 286
39 26 538
40 2 185
5 20 787
5 2 92
34 26 12
24 18 368
47 29 305
13 59 4
37 22 597
12 11 132
23 31 485
37 53 175
26 6 571
6 35 242
50 36 218
51 6 606
32 51 122
51 22 392
39 0 605
10 3 647
7 51 227
16 25 671
16 55 559
3 10 353
13 51 268
26 51 511
0 20 165
10 6 493
15 21 37
12 3 641
26 36 115
7 17 598
2 56 141
7 20 411
32 10 586
35 14 412
3 21 653
39 56 355
26 6 417
35 11 172
10 39 215
43 37 676
32 20 103
6 14 747
47 5 130
23 7 564
35 62 637
12 6 353
33 10 405
10 40 287
25 14 508
23 28 165
36 14 281
36 21 426
34 55 323
33 7 415
7 36 626
35 40 32
26 50 420
51 36 341
6 25 217
51 14 413
10 62 16
4 7 593
4 25 411
41 18 224
6 36 728
38 44 231
37 14 71
33 62 439
33 39 462
23 28 40
41 14 421
13 2 166
13 23 202
23 16 701
37 63 711
44 55 571
46 46 572
37 2 567
27 13 359
23 25 47
10 51 332
34 16 424
23 0 457
2 51 553
34 14 255
23 62 709
23 19 473
27 32 311
23 7 184
26 25 415
50 28 687
5 50 505
0 28 188
43 46 505
46 31 701
34 23 569
26 28 424
40 56 553
33 56 459
7 8 678
6 57 740
23 39 21
5 5 92
25 62 451
43 55 473
23 13 240
32 28 292
5 13 627
45 51 528
33 25 215
33 32 716
2 16 613
17 26 580
2 63 160
2 7 137
33 17 134
51 51 155
38 40 726
23 18 670
27 49 40
16 13 755
2 37 749
13 63 735
0 28 439
43 17 49
4 13 218
10 20 531
0 40 13
23 17 396
3 39 565
35 63 671
28 46 538
36 57 430
47 49 138
51 51 593
0 41 501
20 23 124
37 0 563
25 59 427
33 63 264
39 2 557
33 13 474
37 14 656
11 44 729
46 46 658
34 29 344
5 26 210
7 28 300
26 62 46
23 11 267
37 34 178
6 51 614
41 31 303
24 35 436
50 59 464
0 32 144
10 11 260
28 9 362
9 40 440
44 14 377
51 31 704
2 14 407
46 18 212
36 11 49
36 1 570
26 46 307
34 42 134
36 39 56
13 36 557
8 49 585
7 50 776
10 62 740
41 11 16
13 42 560
50 42 783
12 11 490
23 37 341
12 1 320
9 20 323
39 51 349
0 29 790
5 7 148
20 49 417
36 56 509
16 16 362
2 59 24
0 9 766
50 5 24
17 63 119
5 16 598
9 40 489
47 17 169
3 9 15
23 20 607
25 51 643
45 40 405
36 49 581
50 26 451
45 28 358
22 13 2
36 13 670
20 62 768
43 39 31
43 11 328
33 16 405
5 13 621
47 23 623
5 22 697
28 28 240
47 41 66
39 59 372
36 8 680
1 3 788
41 63 248
33 11 770
33 37 265
51 34 362
32 16 126
0 46 280
24 61 768
12 12 577
26 49 589
32 13 521
33 46 676
23 9 613
5 40 213
8 46 5
33 36 611
3 52 114
0 55 640
13 46 86
33 49 622
24 31 344
51 52 236
33 10 431
41 37 371
23 28 171
32 1 758
26 44 205
22 31 190
2 32 393
23 0 127
5 39 368
23 58 751
34 51 528
33 22 58
2 29 121
51 56 610
51 28 501
24 36 165
7 51 193
50 59 156
5 63 549
23 51 63
38 51 631
7 63 605
4 22 473
34 49 288
24 51 703
43 25 639
50 49 237
23 52 654
4 22 312
24 57 689
5 50 207
13 62 123
3 12 407
37 39 399
16 19 609
24 28 562
0 35 26
27 31 560
7 49 704
2 49 775
13 16 455
34 53 146
44 52 726
40 56 202
23 50 149
26 23 92
51 12 591
26 32 782
18 10 181
0 18 567
6 14 466
39 31 583
1 47 763
15 20 454
7 25 201
45 11 214
12 5 790
39 17 355
33 19 728
23 7 57
13 39 81
33 35 276
4 3 43
23 35 478
24 7 718
35 60 477
50 6 313
45 46 165
