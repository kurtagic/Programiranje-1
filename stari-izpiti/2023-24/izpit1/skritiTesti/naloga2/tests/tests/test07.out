[605, 758, 589, 788, 0, 790, 766, 769, 680, 766, 679, 770, 662, 755, 784, 0, 730, 662, 670, 761, 787, 653, 785, 785, 0, 784, 580, 0, 790, 790, 0, 704, 782, 786, 602, 783, 728, 749, 0, 786, 751, 741, 783, 0, 761, 0, 740, 786, 0, 775, 776, 780, 726, 534, 0, 789, 780, 740, 751, 732, 755, 768, 791, 790]
